crates/ra_assists/src/lib.rs

```diff
@@ -74,11 +75,11 @@ where
             .iter()
             .filter_map(|f| f(ctx.clone()))
             .map(|a| match a {
-                Assist::Resolved { label, action } => (label, action),
+                Assist::Resolved { label, available_actions } => (label, available_actions),
                 Assist::Unresolved { .. } => unreachable!(),
             })
             .collect::<Vec<_>>();
-        a.sort_by(|a, b| match (a.1.target, b.1.target) {
+        a.sort_by(|a, b| match (a.1[0].target, b.1[0].target) {
```

Comment:
Here we order assits by its first action only. I think it's ok even when assists with multiple actions appear, but it's better to note that anyway.